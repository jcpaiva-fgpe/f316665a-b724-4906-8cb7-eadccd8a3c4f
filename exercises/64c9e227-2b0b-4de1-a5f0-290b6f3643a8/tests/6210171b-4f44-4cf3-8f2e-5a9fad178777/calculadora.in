20
1691
1661
347
0
11
222
3333
44444
555555
6666666
77777777
888888888
9999999999
6012568952109
096695105916501569960
096695105976501569960
80608881965856580909590688906560608595859618880908
2826521012510085985215510565825095508619869862969966962986986198055605285950155125865800152101259282
15225168051626268850580668968100610822208121885256062006591290051801829611509600105185520629205181251182190050928212862506526660865962981055825228682920185808582926912996286800128210225585111281681860265921965892596050521520605580682866829985211696896896911258662899828908550902512505096526859612659209818918211158552201282100898296621692628580858102628982252855018629659809992590529821282605006128115218150262902558150100960511962810815006216590029095258812180222801900189689908505889292915089152251
15225168051626268850580668968100610822208121885256062006591290051801829611509600105185520629205181251182190050928212862506526660865962981055825228682920185808582926912996286800128210225585111281681860265921965892596050521520605580682866829985211696896896911258662899828908550902512505096526859612659209818918211158552201282100898296621692628580858102628982252855018629659809992590529821282605006128115218150262902558150100960511962810815006216590029095258812180222801900189689908505889292915089152253
