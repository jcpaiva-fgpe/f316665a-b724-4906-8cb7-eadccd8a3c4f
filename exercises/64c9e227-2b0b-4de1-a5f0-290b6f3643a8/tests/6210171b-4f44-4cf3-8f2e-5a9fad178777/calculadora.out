SIM
NAO
NAO
SIM
SIM
SIM
NAO
NAO
SIM
NAO
NAO
SIM
NAO
SIM
SIM
NAO
SIM
SIM
SIM
NAO
